examples/policies/ngx-example/1.0.0/apicast-policy.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+  "name": "Ngx example policy",
+  "description":
+  ["This policy is meant to be just an example.",
+    "It sets request headers based on the configuration.",
+    "And prints a log entry for each header set. " ],
+  "version": "0.1",
+  "configuration": {
+    "type": "object",
+    "additionalProperties": {
```

Comment:
Maybe this would be easier to understand if we made the fields explicit by specifying a `set_header` object with two properties: `header` and `value`. What do you think?

Reply:
True. That is what I wanted to do :)  I might need to change it to an array, but that is probably for the better.

Reply:
Changed to an array.